### PR TITLE
Add numeric and new data update, please double check

### PR DESCRIPTION
**Pull request by topherhuebel**

### CODE REVIEW

File: PANCHANGE_data_preprocessing2020-07-15.Rmd [hunk 1]
```diff
@@ -374,15 +394,9 @@ glad.gad.raw.id <- glad.gad.raw %>%
...
     gad.sit_restless_hard_prepan = gad.sit_restless_hard,
     gad.becoming_easily_annoyed_or_irritable_prepan = gad.becoming_easily_annoyed_or_irritable,
-    gad.awful_feeling_afraid_happen_prepan = gad.awful_feeling_afraid_happen,
-    gad.feeling_nervous_anxious_or_on_edge_prepan_numeric = gad.feeling_nervous_anxious_or_on_edge_numeric,
-    gad.control_worrying_stop_prepan_numeric = gad.control_worrying_stop_numeric,
-    gad.worrying_too_much_about_different_things_prepan_numeric = gad.worrying_too_much_about_different_things_numeric,
-    gad.trouble_relaxing_prepan_numeric = gad.trouble_relaxing_numeric,
-    gad.sit_restless_hard_prepan_numeric = gad.sit_restless_hard_numeric,
-    gad.becoming_easily_annoyed_or_irritable_prepan_numeric = gad.becoming_easily_annoyed_or_irritable_numeric,
-    gad.awful_feeling_afraid_happen_prepan_numeric = gad.awful_feeling_afraid_happen_numeric
```
Comment (knthompson26) [inline PANCHANGE_data_preprocessing2020-07-15.Rmd:384]: Why have these been removed but not added back in?

File: PANCHANGE_data_preprocessing2020-07-15.Rmd [hunk 2]
```diff
@@ -643,42 +660,78 @@ glad.mhq.raw.id <- glad.mhq.raw.id %>%
...
+Depression and anxiety comorbidity
+```{r glad mhd depression_anxiety}
+glad.mhd.raw.id <- glad.mhd.raw.id %>%
+  mutate(
+    depression_and_anxiety_numeric =
+      case_when(
+        depressive_disorders_numeric == 1 &
+          anxiety_disorders_numeric == 1 ~ 0,
+        depressive_disorders_numeric == 1 &
+          anxiety_disorders_numeric == 1 ~ 1,
```
Comment (knthompson26) [inline PANCHANGE_data_preprocessing2020-07-15.Rmd]: I think there is a mistake here - they are both saying that depressive & anxiety disorders == 1

File: PANCHANGE_data_preprocessing2020-07-15.Rmd [hunk 3]
```diff
@@ -643,42 +660,78 @@ glad.mhq.raw.id <- glad.mhq.raw.id %>%
...
+        depressive_disorders_numeric == 0 &
+          anxiety_disorders_numeric == 1 ~ 3
+      )
   )
+
+glad.mhd.raw.id <- glad.mhd.raw.id %>%
+  mutate(
+    depression_and_anxiety =
+      recode_factor(depression_and_anxiety_numeric,
+                    "0" = "No depressive or anxiety disorder",
```
Comment (knthompson26) [inline PANCHANGE_data_preprocessing2020-07-15.Rmd:739]: This category is wrong above - they should both ==0

**Review by knthompson26 — CHANGES_REQUESTED** [review]
There is one of the case_when where the logic is wrong - I have commented on it here. Once you've changed this then I can approve!

File: PANCHANGE_data_preprocessing2020-07-15.Rmd (hunk 1 above)
Comment (knthompson26) [reply PANCHANGE_data_preprocessing2020-07-15.Rmd:384]: I realise now that these are the numeric ones! Ignore the above comment!

Comment (topherhuebel) [reply PANCHANGE_data_preprocessing2020-07-15.Rmd:384]: but wait.. there is one line too many removed

File: PANCHANGE_data_preprocessing2020-07-15.Rmd (hunk 2 above)
Comment (topherhuebel) [reply PANCHANGE_data_preprocessing2020-07-15.Rmd]: changed

**Review by knthompson26 — APPROVED** [review]

**Pull request merged**